inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -481,9 +482,9 @@ void MKLDNNROIPoolingNode::execute() {
         IE_THROW() << "CPU ROI Pooling node with name '" << getName() << "' doesn't have primitive descriptors.";
     auto config = selectedPrimitiveDescriptor->getConfig();
 
-    auto src_strides = config.inConfs[0].desc.getBlockingDesc().getStrides();
-    auto dst_strides = config.outConfs[0].desc.getBlockingDesc().getStrides();
-    size_t src_roi_step = config.inConfs[1].desc.getBlockingDesc().getStrides()[0];
+    auto src_strides = MemoryDescUtils::convertToBlockedDescriptor(*config.inConfs[0].desc).getStrides();
+    auto dst_strides = MemoryDescUtils::convertToBlockedDescriptor(*config.outConfs[0].desc).getStrides();
+    size_t src_roi_step = MemoryDescUtils::convertToBlockedDescriptor(*config.inConfs[1].desc).getStrides()[0];
```

Comment:
At the moment we consider using MemoryDesc form edge memory at the execute stage.

Reply:
fixed